sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -34,70 +36,132 @@ describe("SmsClient", async () => {
     }
   });
 
-  it("sends a SMS message to multiple recipients", async () => {
-    const connectionString = env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING as string;
+  //helper functions
+  const expectCorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 202);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.isString(actualSmsResult.messageId);
+    assert.isTrue(actualSmsResult.successful);
+    assert.notExists(actualSmsResult.errorMessage, "no error message for success");
+  };
+
+  const expectIncorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 400);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.notExists(actualSmsResult.messageId, "no message id for errors");
+    assert.isFalse(actualSmsResult.successful);
+    assert.exists(actualSmsResult.errorMessage);
+  };
+
+  it("can send a SMS message", async () => {
     const fromNumber = env.AZURE_PHONE_NUMBER as string;
     const validToNumber = env.AZURE_PHONE_NUMBER as string;
-    const invalidToNumber = "+18005551234567";
+    const results = await smsClient.send({
+      from: fromNumber,
+      to: [validToNumber],
+      message: "test message"
+    });
 
-    const recipients = [validToNumber, invalidToNumber];
+    assert.lengthOf(results, 1, "must return as many results as there were recipients");
+    expectCorrectResult(results[0], validToNumber);
+  });
 
-    const smsClient = new SmsClient(connectionString);
+  it("can send a SMS message with options passed in", async () => {
+    const fromNumber = env.AZURE_PHONE_NUMBER as string;
+    const validToNumber = env.AZURE_PHONE_NUMBER as string;
+    const results = await smsClient.send(
+      {
+        from: fromNumber,
+        to: [validToNumber],
+        message: "test message"
+      },
+      {
+        enableDeliveryReport: true,
+        tag: "SMS_LIVE_TEST"
+      }
+    );
 
-    const request: SmsSendRequest = {
+    assert.lengthOf(results, 1, "must return as many results as there were recipients");
+    expectCorrectResult(results[0], validToNumber);
+  });
+
+  it("sends a new message each time send is called", async () => {
+    const fromNumber = env.AZURE_PHONE_NUMBER as string;
+    const validToNumber = env.AZURE_PHONE_NUMBER as string;
+
+    const sendRequest: SmsSendRequest = {
       from: fromNumber,
-      to: recipients,
+      to: [validToNumber],
       message: "test message"
     };
-
     const options: SmsSendOptions = {
       enableDeliveryReport: true,
       tag: "SMS_LIVE_TEST"
     };
 
-    const results = await smsClient.send(request, options);
+    const firstResults = await smsClient.send(sendRequest, options);
+    const secondResults = await smsClient.send(sendRequest, options);
```

Comment:
Lets also check if both these results have the expected fields. I think we have that in the excel sheet.

Reply:
can do, i left it out because i thought it might've been overly redundant